students/rachel_s/session05/mailroom.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+
+import csv
+import os
+import datetime
+import sys
+from collections import defaultdict
+
+# Import donor data
+def data_import(filename):
+    donor_info = defaultdict(list)
+    with open(filename) as csv_file:
+        csv_reader = csv.reader(csv_file, delimiter=',')
+        for line in csv_reader:
+            donor_info[line[0].lower()].append(int(line[1]))
```

Comment:
Looking for where you applied any list/dict/set comprehension, and not sure if I found any.

I thought this might be a good place to apply list comprehension, but maybe not... Since the assignment requires some list comprehension, could you try adding some? I know it feels like making up, but it's for practice...